travvy/src/components/AttractionDetails.vue

```diff
@@ -29,4 +44,50 @@
 #nav a.router-link-exact-active {
   color: #42b983;
 }
+
+.back {
+  display: inline-block;
+  border-radius: 4px;
+  background-color: #FEC195;
+  border: none;
+  color: #FFFFFF;
+  text-align: center;
+  font-size: 22px;
+  padding: 12px 22px;
+  width: 150px;
+  transition: all 0.5s;
+  cursor: pointer;
+  margin: 5px;
+  margin-left: -1690px;
+  background: linear-gradient(
+      to bottom,
+        rgba(255,95,0) 8%,
+        rgba(254,193,149) 56%,
+        rgba(240,228,216) 82%,
+        rgba(255,242,225) 100%)
+}
+.back span {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  transition: 0.5s;
+}
+
+.back span:after {
```

Comment:
I see you added button functionality/behavior when the user hovers, similar to the home page buttons!! nice touch :)

Reply:
Thanks!! Just wanted to make it cohesive throughout the platform!! Also added a gradient for fun :)